extensions.json

```diff
@@ -585,10 +585,8 @@
     },
     {
       "id": "ms-python.python",
-      "repository": "https://github.com/Microsoft/vscode-python",
       "version": "2020.7.1",
```

Comment:
2020.7.96456 ?

Reply:
Their tag names are inconsistent with the versions, see https://github.com/microsoft/vscode-python/releases/tag/2020.7.96456

Reply:
they don't seem to use 2020.7.1 on marketplace: https://marketplace.visualstudio.com/items?itemName=ms-python.python It was confusing to me to understand whether `2020.7.1` corresponds to MS marketplace version or not or to which commit in the repo.

Reply:
Strange, maybe the last part is replaced in their release process. Switching to prebuilt packages should solve this.

Reply:
Keeping `2020.7.1` here will skip publishing (because `2020.7.1` is already published).

I suggest removing the `version` entirely (it's just an optimization to avoid a pointless publishing attempt, removing it guarantees publishing is at least attempted):

```suggestion
```